src/NServiceBus.AcceptanceTests/Recoverability/When_message_is_deferred_by_delayed_retries_using_dtc.cs

```diff
@@ -23,11 +24,10 @@ public class When_message_is_deferred_by_delayed_retries_using_dtc : NServiceBus
                         Id = c.Id
                     }))
                  )
-                .WithEndpoint<ErrorSpy>()
-                .Done(c => c.MessageMovedToErrorQueue)
+                .Done(c => c.FailedMessages.Any())
                 .Run();
 
-            Assert.Greater(context.NumberOfProcessingAttempts, 1, "Should retry at least once");
+            Assert.Greater(context.NumberOfProcessingAttempts, 3, "Should retry at least once");
```

Comment:
Why are we testing for > 3 now, and how does that mean "Should retry at least once" ?